### PR TITLE
Update country display names

### PR DESCRIPTION
**Pull request by Corniel**
There was no code in place (anymore) that allowed to (re)generate the resource files with the display names, nor a way to detect that the ISO-3166-1 standard had been adjusted otherwise (countries added, or removed).

This PR includes code to generate the `Country` constants, Dutch, German, and English display names. (And a small bug fix when parsing countries that have an ISO code that has previously by owned by a now former country).

### CODE REVIEW

File: specs/Qowaiv.Specs/TestTools/_specs/Have_type_converter_definded_specs.cs
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/TestTools/_specs/Have_type_converter_definded_specs.cs:1]: defined

File: src/Qowaiv/Globalization/CountryConstants.cs [hunk 1]
```diff
@@ -92,7 +92,7 @@ namespace Qowaiv.Globalization;
     /// <summary>Describes the country Bolivia (BO).</summary>
     public static readonly Country BO = new("BO");
 
-    /// <summary>Describes the country Bonaire (BQ).</summary>
+    /// <summary>Describes the country Caribbean Netherlands (BQ).</summary>
```
Comment (LauraXiulan) [inline src/Qowaiv/Globalization/CountryConstants.cs:95]: Is this right? Earlier it was the combination of Bonaire, Sint Eustasius and Saba

File: src/Qowaiv/Globalization/CountryConstants.cs [hunk 2]
```diff
@@ -380,7 +380,7 @@ namespace Qowaiv.Globalization;
     /// <summary>Describes the country Kazakhstan (KZ).</summary>
     public static readonly Country KZ = new("KZ");
 
-    /// <summary>Describes the country Lao People'S Democratic Republic (LA).</summary>
+    /// <summary>Describes the country Loas (LA).</summary>
```
Comment (LauraXiulan) [inline src/Qowaiv/Globalization/CountryConstants.cs:383]: Laos

**Review by LauraXiulan — CHANGES_REQUESTED** [review]

File: specs/Qowaiv.Specs/TestTools/Wikipedia/WikiQuery.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        Url = new Uri($"https://{language}.wikipedia.org/w/api.php?action=query&prop=revisions&titles={lemma}&rvprop=content&rvslots=*&format=json");
+        Cached = new(Path.Combine(Path.GetTempPath(), "WikiCache", $"{ToFile(lemma)}.{language}.json"));
+        if (!Cached.Directory!.Exists)
+        {
+            Cached.Directory.Create();
+        }
+    }
+
+    private static string ToFile(string lemma)
```
[the remark this answers is not in the document]
Comment (Corniel) [reply specs/Qowaiv.Specs/TestTools/Wikipedia/WikiQuery.cs:15]: Query should be removed. Well spotted!

File: specs/Qowaiv.Specs/Globalization/Countries_generation.cs [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+    // Overrides of Wikipedia: 
+    private static readonly Dictionary<string, string> Shorten = new()
+    {
+        ["Bolivia (Plurinational State of)"] = "Bolivia",
+        ["Bonaire, Sint Eustatius and Saba"] = "Caribbean Netherlands",
+        ["Iran (Islamic Republic of)"] = "Iran",
+        ["Korea (Democratic People's Republic of)"] = "North Korea",
+        ["Korea, Republic of"] = "South Korea",
+        ["Lao People's Democratic Republic"] = "Loas",
```
[the remark this answers is not in the document]
Comment (Corniel) [reply specs/Qowaiv.Specs/Globalization/Countries_generation.cs:196]: Oops.

File: src/Qowaiv/Globalization/CountryConstants.cs (hunk 1 above)
Comment (Corniel) [reply src/Qowaiv/Globalization/CountryConstants.cs:95]: Caribbean Netherlands is the combination of Bonaire, Sint Eustasius and Saba, so yes. Unless you have a source that explains that is wrong?!

Comment (LauraXiulan) [reply src/Qowaiv/Globalization/CountryConstants.cs:95]: It is more that the Caribbean Netherlands is used to describe the collective of those three, should it then also be used to describe one of the three?

File: specs/Qowaiv.Specs/TestTools/Solution.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Qowaiv.TestTools;
+
+public static class Solution
```
[the remark this answers is not in the document]
Comment (Corniel) [reply specs/Qowaiv.Specs/TestTools/Solution.cs:3]: Nothing in itself. Is helper code that allows to say: `Solution.Root.File(path)`. This is used with the code generation, and the paths to resolve.

File: src/Qowaiv/Globalization/CountryConstants.cs (hunk 1 above)
Comment (Corniel) [reply src/Qowaiv/Globalization/CountryConstants.cs:95]: BQ is for Bonaire, Sint Eustasius and Saba, not just Bonaire.

File: specs/Qowaiv.Specs/TestTools/Resx/XResourceFileHeader.cs [hunk 6]
```diff
@@ -1,17 +1,14 @@
...
 namespace Qowaiv.TestTools.Resx;
 
 /// <summary>Represents a header of a resource file.</summary>
 [Serializable]
 [DebuggerDisplay("Header, Name: {Name}, Value: '{Value}'")]
-	public sealed class XResourceFileHeader : IXmlSerializable
-	{
-		/// <summary>Gets the Resourse mime type header (text/microsoft-resx).</summary>
-		public static readonly XResourceFileHeader ResMimeType = new("resmimetype", "text/microsoft-resx");
+public sealed class XResourceFileHeader //: IXmlSerializable
```
Comment (LauraXiulan) [inline specs/Qowaiv.Specs/TestTools/Resx/XResourceFileHeader.cs:8]: // should not be here I think

**Review by LauraXiulan — APPROVED** [review]

**Pull request merged**